hw/ip/otbn/data/otbn.hjson

```diff
@@ -102,30 +102,76 @@
         }
       ]
     } // register : status
-    { name: "ERR_CODE",
-      desc: "Error Code",
+    { name: "ERR_BITS",
+      desc: '''
+        Error bitfield, reads as non-zero if an error was seen during OTBN
+        operation
+      ''',
       swaccess: "ro",
       hwaccess: "hwo",
       fields: [
-        { bits: "31:0",
-          name: "err_code",
+        { bits: "0",
+          name: "bad_data_addr"
           desc: '''
-            The error cause if an error occurred.
-
-            This is set to zero (ErrCodeNoError) when OTBN successfully
-            completes an operation or a positive value when OTBN stops on an
-            error.
-
-            While OTBN is running (!!CMD.busy set), this has no meaning. When
-            OTBN stops this will be set to indicate the outcome. It is
-            guaranteed not to change until OTBN is run again (with !!CMD.start).
-
-            See <a href='#error-conditions'>Error Conditions</a> for
-            details on the possible values.
+            A DMEM read or write occurred with an out of bounds or unaligned
+            address.
           '''
         }
+        { bits: "1",
+          name: "bad_insn_addr"
+          desc: '''
+            A DMEM read or write occurred with an out of bounds or unaligned
+            address.
+          '''
+        }
+        { bits: "2",
+          name: "call_stack"
+          desc: '''
+            A instruction tried to pop from an empty call stack or push to a
+            full call stack.
+          '''
+        }
+        { bits: "3",
+          name: "illegal_insn"
+          desc: '''
+            <ul>
```

Comment:
For these lists, can we prefix with something like "One of the following happened"? Otherwise, a quick glance might suggest that all of them need to happen.